app/assess/data.py

```diff
@@ -342,3 +346,34 @@ def get_comments(application_id: str, sub_criteria_id: str):
     )
     comment_response = get_data(comment_endpoint)
     return comment_response
+
+def get_file_url(filename: str, application_id: str):
+    """_summary_: Function is set up to retrieve
+    files from aws bucket.
+    Args:
+        filename: Takes an filename, application_id: Takes an application_id # noqa
```

Comment:
Break this onto two lines maybe? Did black wrap it? Maybe remove the comma if so?